pkg/driverbuilder/builder/builders.go

```diff
@@ -415,7 +421,6 @@ func GetResolvingURLs(urls []string) ([]string, error) {
 		}
 		if res.StatusCode == http.StatusOK {
 			results = append(results, u)
-			slog.With("url", u).Debug("kernel header url found")
```

Comment:
Again, passing the printer down here would be a bit troublesome.
I preferred to just kill this debug log (and adding a log in `KernelDownloadScript`: https://github.com/falcosecurity/driverkit/pull/333/files#diff-16a66a3bc20a8ec693f64ed821148f71efa386a7e942444444eb591512bf129bR173)